lib/chf/data_fixes/work_dates.rb

```diff
@@ -0,0 +1,26 @@
+module CHF
+  module DataFixes
+    class WorkDates
+      attr_reader :work
+
+      def initialize(work)
+        @work = work
+      end
+
+      # possibly makes changes to work (without saving), returns true
+      # if changes were made, so caller can do expensive save.
+      def change
+        updated = false
```

Comment:
There is a `.changed?` that I was just messing around with but it doesn't appear to be universally useful:

```
[21] pry(main)> w.visibility
=> "open"
[22] pry(main)> w.visibility = "restricted"
=> "restricted"
[23] pry(main)> w.changed?
=> false
[24] pry(main)> w.title = ["hello"]
=> ["hello"]
[25] pry(main)> w.changed?
=> true
```